pkg/process/config/yaml_config.go

```diff
@@ -56,6 +56,13 @@ func (a *AgentConfig) LoadAgentConfig(path string) error {
 	}
 	a.CheckIntervals[DiscoveryCheckName] = discoveryInterval
 
+	eventsInterval := config.Datadog.GetDuration("process_config.event_collection.interval")
+	if eventsInterval <= 0 {
```

Comment:
A common mistake is to omit units from a duration in a config, which go would unhelpfully interpret as nanoseconds. Would it make sense to check for a minimum here, say 1s?